scanner/identifier_test.go

```diff
@@ -0,0 +1,66 @@
+package scanner_test
+
+import (
+	"testing"
+
+	"github.com/grafana/river/scanner"
+	"github.com/stretchr/testify/require"
+)
+
+func TestIsValidIdentifier(t *testing.T) {
```

Comment:
Can we add fuzz tests for SanitizeIdentifier? The fuzz test should verify the sanitized output is a valid identifier.